icons/tree-deciduous.svg

```diff
@@ -9,6 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M8 19h8a4 4 0 0 0 3.8-2.8 4 4 0 0 0-1.6-4.5c1-1.1 1-2.7.4-4-.7-1.2-2.2-2-3.6-1.7a3 3 0 0 0-3-3 3 3 0 0 0-3 3c-1.4-.2-2.9.5-3.6 1.7-.7 1.3-.5 2.9.4 4a4 4 0 0 0-1.6 4.5A4 4 0 0 0 8 19Z" />
+  <path d="M5.8 11.7A4 4 0 0 0 8 19h8a4 4 0 0 0 2.2-7.3A3.5 3.5 0 0 0 15 6a3 3 0 1 0-6 0 3.5 3.5 0 0 0-3.2 5.7" />
```

Comment:
This one here is closer to my original intent and keeps the path perfectly aligned to the grid (yours has a loss of .12px horizontally):
```suggestion
  <path d="M8 19a4 4 0 0 1-2.24-7.32A3.5 3.5 0 0 1 9 6.03V6a3 3 0 1 1 6 0v.04a3.5 3.5 0 0 1 3.24 5.65A4 4 0 0 1 16 19Z" />
```